scripts/preview-theme.js

```diff
@@ -44,6 +44,11 @@ const REQUIRED_COLOR_PROPS = ACCEPTED_COLOR_PROPS.slice(0, 4);
 const INVALID_REVIEW_COMMENT = (commentUrl) =>
   `Some themes are invalid. See the [Automated Theme Preview](${commentUrl}) comment above for more information.`;
 
+// Retrieve octokit instance.
+const OCTOKIT = github.getOctokit(getGithubToken());
+const pullRequestId = prNumber ? prNumber : getPrNumber();
```

Comment:
Maybe we should here also use the CONST convention?

```suggestion
const PULL_REQUEST_ID = prNumber ? prNumber : getPrNumber();
```